### PR TITLE
TsFile V4 for Table Model

### PR DESCRIPTION
**Pull request by jt2594838**

### CODE REVIEW

File: .github/workflows/unit-test.yml [hunk 1]
```diff
@@ -77,6 +77,11 @@ jobs:
               core.setOutput('platform_suffix', ``)
             }
 
+      - name: Install python packages
+        if: ${{ matrix.os == 'ubuntu-latest'}}
+        run: pip3 install cython pandas numpy
+
+
```
Comment (HTHou) [inline .github/workflows/unit-test.yml]: This part is useless anymore and can be removed.

File: cpp/pom.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.apache.tsfile</groupId>
         <artifactId>tsfile-parent</artifactId>
-        <version>1.0.1-SNAPSHOT</version>
+        <version>1.0.1-tsfilev4-240801-SNAPSHOT</version>
```
Comment (HTHou) [inline cpp/pom.xml]: Should we keep the version 1.0.1-SNAPSHOT? We can change it later after the PR merged.

File: .github/workflows/unit-test.yml (hunk 1 above)
Comment (jt2594838) [reply .github/workflows/unit-test.yml]: Removed

File: cpp/pom.xml (hunk 2 above)
Comment (jt2594838) [reply cpp/pom.xml:25]: This PR is just to trigger the CI, and I will not merge it into 'develop'.
Actually, I will merge it back to tsfile_v4.

Comment (jt2594838) [reply cpp/pom.xml]: How about we update it to 1.2.0-SNAPSHOT now?

**Review by JackieTien97 — APPROVED** [review]

**Review by HTHou — APPROVED** [review]

**Pull request merged**